### PR TITLE
Update animals table

### PR DESCRIPTION
**Pull request by BranBayou**
### In this project I did the following
- added a column species of type string to my animals table. 
- modified my schema.sql file.
- added 7 more data to my animals table.
- Inside a transaction
     - updated the animals table by setting the species column to unspecified.
     - update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
     - updated the animals table by setting the species column to pokemon for all animals that don't have species already set.
- created a savepoint for the transaction.
- rollback to the savepoint
- wrote queries to answer the following questions:
    How many animals are there?
    How many animals have never tried to escape?
    What is the average weight of animals?
    Who escapes the most, neutered or not neutered animals?
    What is the minimum and maximum weight of each type of animal?
    What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Screenshot-1
Screenshot from 2023-06-21 18-03-49
Screenshot-2
Screenshot from 2023-06-21 18-07-32
Screenshot-3
Screenshot from 2023-06-21 17-52-30
Screenshot-4
Screenshot from 2023-06-21 17-57-26
Screenshot-5
Screenshot from 2023-06-21 20-20-42
Screenshot-6
Screenshot from 2023-06-21 20-22-25
Screenshot-7
Screenshot from 2023-06-21 20-28-38
Screenshot-8
Screenshot from 2023-06-21 20-31-35
Screenshot-9
Screenshot from 2023-06-21 20-36-51
Screenshot-10
Screenshot from 2023-06-21 20-38-14
Screenshot-11
Screenshot from 2023-06-21 20-40-09
Screenshot-12
Screenshot from 2023-06-21 20-43-31
Screenshot-13
Screenshot from 2023-06-21 20-44-38
Screenshot-14
Screenshot from 2023-06-21 20-45-26
Screenshot-15
Screenshot from 2023-06-21 20-59-24
Screenshot-16
Screenshot from 2023-06-21 21-02-46
Screenshot-17
Screenshot from 2023-06-21 23-26-12

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @BranBayou 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- The information given for the Install, Usage and Run Tests subsections of the Getting Started sections are that of a Ruby on Rails project and not a Postgresql project. Kindly change them. 👍 

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -7,4 +7,70 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+
+SELECT species, 
+MIN(weight_kg), MAX(weight_kg)
+FROM animals
+GROUP BY species;
+
+SELECT species,
+AVG(escape_attempts)
+FROM animals
+WHERE date_of_birth BETWEEN '1990-01-01' AND '1999-12-31'
```
Comment (rloterh) [inline queries.sql]: - [ ] In creating a query for the question: _**What is the average number of escape attempts per animal type of those born between 1990 and 2000?**_ our query should look like:
```sql
   SELECT species, AVG(escape_attempts) FROM animals
   GROUP BY species
   WHERE  date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/12/31'
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Bran, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!
🚦 __🏃 🏡 🏃‍♂️ 🏃‍♀️ 🏚️ 🌴

## Highlights 👍  
- Good job so far implementing the update query!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (BranBayou) [reply queries.sql]: Hi @rloterh,  thank you for your constructive comments.

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @BranBayou, 

To highlight 👍🏽 
✔️ SQL commands are added
✔️ Queries are successful 
✔️ Screenshots are added

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**